Tasks/Xcode/Tests/L0.ts

```diff
@@ -458,9 +458,94 @@ describe('Xcode L0 Suite', function () {
             '-exportPath /user/build/_XcodeTaskExport_testScheme -exportOptionsPlist _XcodeTaskExportOptions.plist'),
             'xcodebuild exportArchive should have been run with -allowProvisioningUpdates to export the IPA from the .xcarchive');
 
+        assert(tr.ran("/usr/libexec/PlistBuddy -c Add iCloudContainerEnvironment string Production _XcodeTaskExportOptions.plist") === false,
+            'PlistBuddy add cloud entitlement should not run');
```

Comment:
People may overlook === false on the end; `!tr.ran` would be more obvious.

BTW, you typically don't need to check that something didn't run. I'd expect the test to fail if it exec'd a command line not listed in the exec section.

Reply:
Right, removing this line.